lib/ui/index.js

```diff
@@ -79,6 +79,10 @@ UI.prototype.writeLine = function(data, writeLevel) {
   if (this.writeLevelVisible(writeLevel)) {
     this.write(data + EOL);
   }
+
+  if (/ERROR/i.match(writeLevel)) {
```

Comment:
Honestly, I'm not sure.  My gut instinct would be that we would only want to log to one or the other, not both. We should just research what is "normal" in *nix land and do that.